app/views/admin/debate_outcomes/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= form_for @petition.debate_outcome, url: admin_petition_debate_outcome_path(@petition), method: :patch do |f| -%>
+
+  <%= render 'admin/petitions/petition_details', f: f %>
+
+  <h2>Debate Outcome</h2>
+
+  <%= form_row :for => [f.object, :debated_on] do %>
+    <%= f.label :debated_on, class: 'form-label' %>
+    <%= error_messages_for_field f.object, :debated_on %>
+    <%= f.text_field :debated_on, type: 'date', tabindex: increment, class: 'form-control' %>
```

Comment:
You could use `date_field` instead? 

http://edgeapi.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-date_field